CMakeLists.txt

```diff
@@ -16,6 +16,7 @@ message("\n=========== mediasoup-broadcaster-demo Build Configuration ==========
 message(STATUS "LIBWEBRTC_INCLUDE_PATH : " ${LIBWEBRTC_INCLUDE_PATH})
 message(STATUS "LIBWEBRTC_BINARY_PATH  : " ${LIBWEBRTC_BINARY_PATH})
 if(NOT APPLE)
+    message(STATUS "OPENSSL_INCLUDE_DIR    : " ${OPENSSL_INCLUDE_DIR})
```

Comment:
open ssl is needed only if it compiled on linux.
on mac os x it uses the framework Security and libcpr+libcurl installed via brew

Reply:
I expected a more detailed and longer description. There are many changes in this PR and we are the ones that will have to maintain them in the future. I'll ask @jmillan about those changes but for now I don't fully understand what they do.

Reply:
Do you want a new PR with only one commit?
Maybe it is more comfortable to manage it.
Let me know